feat: Navigation nodes with a page attached use the preview url in the edit or preview mode. (#131)
FidelityInternational/djangocms-navigation: CHANGELOG.rst

Unreleased
==========

> * feat: The navigation will output the preview url for any navigation nodes that have a page attached in the edit or preview mode.

Author: ```suggestion
* feat: Navigation nodes with a page attached use the preview url in the edit or preview mode. 
```
